NEWS.md

```diff
@@ -1,3 +1,6 @@
+0.21.1 (2018-03-08)
```

Comment:
As we are using [SEMVER](https://semver.org/) we should tag this version as a minor at least so it should be 0.22.0 and don't include yet the day and month of the release so could be possible to add more PRs until we release this :)

Reply:
Cool, will update :) I still get mixed up about semver.